Week2/index.js

```diff
@@ -0,0 +1,127 @@
+const mysql = require('mysql');
+const { promisify } = require('util');
+const inquirer = require('inquirer');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'new_world',
+});
+
+connection.connect();
+
+const runQuery = promisify(connection.query).bind(connection);
+
+// What is the capital of country X ?
+const capitalName = async () => {
+  const ask = [
+    {
+      type: 'input',
+      name: 'countryName',
+      message: 'Country name?',
+    },
+  ];
+  const { countryName } = await inquirer.prompt(question);
```

Comment:
The program is not running, I am getting error `question is not defined`. Is this the latest version of your homework?